.gitignore

```diff
@@ -1 +1,4 @@
 *.pyc
```

Comment:
Please have a .gitignore in the demo/ios and put the related iOS entries there. Also maybe we should take a look at existing .gitignore of iOS projects like this:
https://gist.github.com/macbaszii/4475403